doc/common/glossary.md

```diff
@@ -20,94 +20,136 @@ To help guide the reader, this glossary provides an introduction to the terminol
-- **Cloud Infrastructure Profile**: The combination of the Cloud Infrastructure Software Profile and the Cloud Infrastructure Hardware Profile that defines the capabilities of the Cloud Infrastructure.
-- **Cloud Infrastructure Software Profile**: defines the behaviour, capabilities and metrics provided by a cloud infrastructure Software Layer
-- **Cloud Infrastructure Software Configuration**: a set of settings (Key:Value) that are applied/mapped to **cloud infrastructure** SW deployment.
+
+- **Hypervisor:** A piece of software with management components that allows the user to partition the underlying physical resources and allocate them to Virtual Machines. Typically, the hypervisor is managed by a cloud management software such as OpenStack.
+
+- **Instance:** is a virtual compute resource, in a known state such as running or suspended, that can be used like a physical server.
+    >_*Note:*_ Can be used to specify VM Instance or Container Instance.
+
+- **Network Function (NF):**  functional block or application within a network infrastructure that has well-defined external interfaces and well-defined functional behaviour.
+  - Within **NFV**, A **Network Function** is implemented in a form of **Virtualised NF** or a **Cloud Native NF**.
+
+
+- **Network Function Virtualisation (NFV):** The concept of separating network functions from the hardware they run on by using a virtual hardware abstraction layer.
+
+- **Network Function Virtualisation Infrastructure (NFVI):** The totality of all hardware and software components used to build the environment in which a set of VAs are deployed (also referred to as cloud infrastructure).
+ >_*Note:*_ The NFVI can span across many locations, e.g. places where data centres or edge nodes are operated. The network providing connectivity between these locations is regarded to be part of the cloud infrastructure. **NFVI** and **VNF** are the top-level conceptual entities in the scope of Network Function Virtualisation. All other components are sub-entities of these two main entities.
+
+- **Network Service (NS):** composition of **Network Function**(s) and/or **Network Service**(s), defined by its functional and behavioural specification, including the service lifecycle.
+
+- **Virtual Application (VA):** A general term for software which can be loaded into a Virtual Machine.
+  >_*Note:*_ a **VNF** is one type of VA.
+
+- **Virtual Machine (VM):** virtualised computation environment that behaves like a physical computer/server.
+  >_*Note:*_ A **VM** consists of all of the components (processor (CPU), memory, storage, interfaces/ports, etc.) of a physical computer/server. It is created using Instance Type together with sizing information or Compute Flavour.  
```

Comment:
Instance Type is not defined - is also used later in another definition

Reply:
Thanks, we have removed the "instance type"